### ISSUE TITLE
Ledger app not supported

### ISSUE THREAD

**Issue by jeffwerner00** [AntelopeIO/Coalition#33]
Ledger app created by Cyberglass is not maintained by anyone

**Comment by jeffwerner00** [comment]
Eric submitted PR, looking for approval

**Comment by jeffwerner00** [comment]
Eric has been engaged with Ledger team and is in process
Eric should validate that the end solution will impact all coalition chains
Can it say "Antelope (EOS, TLOS, WAXP, UTX)"?

**Comment by jeffwerner00** [comment]
Eric waiting on Ledger team for moving forward
Still waiting on answer on adding three other token names
Should we develop a formal business relationship with Ledger (beyond technical)? - Ted/Jeff to pursue

**Comment by jeffwerner00** [comment]
Ledger meeting occurred
- cannot put all tokens into single line: best option recommended is single binary, modified with metadata to create four versions.
- ENF can complete this work if needed; can supply PjM to coordinate the collection of information from chains
- Ledger is requesting a demonstration that the single binary can be utilized with each chain
- Ted to get more info on whether a generic "Antelope" version has constraints
- Jeff to assign a PjM and kick off project (Lukas, G, Justin contacts)

**Comment by jeffwerner00** [comment]
Possible to have a fifth "Antelope" version; looking into search mechanics
ETH model: Main Antelope app with individual chain apps installed

**Comment by tedcahalleos** [comment]
Brian Hazzard now the Product Manager of this initiative.
- New call scheduled with Thomas Julien
- Thomas working to rest long name for Antelope App: EOS-WAXP-TLOS-UTX

**Comment by jeffwerner00** [comment]
Need demo videos from Telos and WAX
Three website / marketing pages needed for each chain - discussing with Ledger on who sets these up

**Comment by jeffwerner00** [comment]
ENF needs to put on hold indefinitely; other chains have option to pickup if needed
